client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -37,13 +37,25 @@
                     <tr v-if="dataset.id">
                         <td>History Content API ID</td>
                         <td>
-                            <div id="dataset-id">{{ dataset.id }} <decoded-id :id="dataset.id" /></div>
+                            <div id="dataset-id">
+                                {{ dataset.id }} <decoded-id :id="dataset.id" /> (<a
+                                    :href="'/tool_runner/rerun?id=' + dataset.id"
+                                    target="_top"
+                                    >rerun</a
+                                >)
+                            </div>
                         </td>
                     </tr>
                     <tr v-if="dataset.history_id">
                         <td>History API ID</td>
                         <td>
-                            <div id="history_id">{{ dataset.history_id }} <decoded-id :id="dataset.history_id" /></div>
+                            <div id="history_id">
+                                {{ dataset.history_id }} <decoded-id :id="dataset.history_id" /> (<a
+                                    :href="'/histories/view?id=' + dataset.history_id"
```

Comment:
I think we want to get away from this direct navigation, but I'm not sure what the replacement is. @dannon is that how you would do this ? Might also be better to have a button for this ?

Reply:
I guess, we kinda have to deal with it for now, not sure about global Vue router status